Added initialLeft / Top to set initial crop region

DangerD256 opened the pull request:
Needed for recropping image is user wants to redo crop.

DangerD256 committed:
Added initialLeft / Top

DangerD256 committed:
Added initialLeft / Top

angekouf reviewed (changes requested):
#Pretty good suggestion and I already downloaded for myself but you removed the crop button... 
Also it would be better if the user could also set a precropped image to load at init.

 @DangerD256 I made a pull request with my changes at your fork.

angekouf committed:
Update croppic.js

Fixing the removed crop button in your versiin... Also allowing the user to set a precropped image to load at init

angekouf commented on croppic.js:
```diff
@@ -401,7 +408,7 @@
 			var cropControlZoomMuchOut =     '';
 			var cropControlRotateLeft =      '';
 	        var cropControlRotateRight =     '';
-	        var cropControlCrop =            '<i class="cropControlCrop"></i>';
```
By emptying the string the crop button is no longer initialized...

DangerD256 committed:
Merge pull request #1 from angekouf/patch-1

Fixing missing crop button and allow the user to set a precropped image
